packages/legacy/src/Plans/components/Wizard/TypeForm.tsx

```diff
@@ -59,11 +58,22 @@ export const TypeForm: React.FunctionComponent<ITypeFormProps> = ({
             <Alert
               variant="warning"
               isInline
-              title="A warm migration is not currently available."
+              title="Warm migration is not currently available."
+            >
+              Warm migration from {PROVIDER_TYPE_NAMES.ovirt} source provider is only supported
+              with a verified secure connection.
+            </Alert>
+          </StackItem>
+        )}
+
+        {isSourceOpenstack && (
+          <StackItem>
+            <Alert
+              variant="warning"
+              isInline
+              title="Warm migration is not currently available."
             >
-              A warm migration from a {PROVIDER_TYPE_NAMES.ovirt} source provider is only supported
-              with a verified secure connection.  To enable a warm migration, first select a secure provider
-              on the <Button variant="link" isInline onClick={() => goToStepById(0)} >General step</Button>.
+              Warm migration from {PROVIDER_TYPE_NAMES.openstack} source provider is unsupported.
```

Comment:
The wording us a bit off.  Suggestions:

  1. "**A** warm migration from **a** openstack source provider is unsupported."
  2. "Warm migration**s** from openstack source provider**s are** unsupported."